### PR TITLE
Misc function matches and improved install.sh

### PR DESCRIPTION
**Pull request by tashi-lp**
Improved install.sh distro compatibility

Matched various functions in `chill_out_7900D0.c` and `5CE690.c`

### CODE REVIEW

File: src/battle/area_flo2/5CE690.c [hunk 1]
```diff
@@ -18,9 +18,19 @@ INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_80218620_5CECB0);
...
-INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_802188EC_5CEF7C);
+ApiStatus func_802188A8_5CEF38(void) {
+    func_8011D82C(1);
+    *D_801512F0 = 1;
+    set_background_color_blend(0, 0, 0, 0);
+    return ApiStatus_DONE2;
+}
+
+ApiStatus func_802188EC_5CEF7C(ScriptInstance* script) {
+    Bytecode* ptrReadPos = script->ptrReadPos;
```
Comment (nanaian) [inline src/battle/area_flo2/5CE690.c:29]: Call this variable `args` pls, they're the arguments given by the script calling this function

File: src/battle/area_flo2/5CE690.c [hunk 2]
```diff
@@ -18,9 +18,19 @@ INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_80218620_5CECB0);
 
 INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_80218774_5CEE04);
 
-INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_802188A8_5CEF38);
-
-INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_802188EC_5CEF7C);
+ApiStatus func_802188A8_5CEF38(void) {
```
Comment (nanaian) [inline src/battle/area_flo2/5CE690.c:21]: The signature of all script-api functions (any returning an `ApiStatus`) should be
```c
ApiStatus func_802188A8_5CEF38(ScriptInstance* script, s32 isInitialCall);
```
regardless of if it uses the args or not (in this case, neither are used).

Comment (nanaian) [reply src/battle/area_flo2/5CE690.c:21]: Ditto for `func_802188EC_5CEF7C`.

File: src/battle/area_flo2/5CE690.c [hunk 3]
```diff
@@ -18,9 +18,19 @@ INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_80218620_5CECB0);
...
+    func_8011D82C(1);
+    *D_801512F0 = 1;
+    set_background_color_blend(0, 0, 0, 0);
+    return ApiStatus_DONE2;
+}
+
+ApiStatus func_802188EC_5CEF7C(ScriptInstance* script) {
+    Bytecode* ptrReadPos = script->ptrReadPos;
+
+    set_background_color_blend(0, 0, 0, get_variable(script, *ptrReadPos++) & 255);
```
Comment (nanaian) [inline src/battle/area_flo2/5CE690.c:31]: This `& 255` looks suspiciously like a cast to `s8` (char). What is the signature of `set_background_color_blend`? If it already takes an s8, this bitwise operation shouldn't be required, and if it doesn't yet, perhaps try changing it to `s8` provided that doesn't break any existing code.

**Review by nanaian — CHANGES_REQUESTED** [review]
Thanks for the PR! Looks great, other than the API function signatures being off.

**Review by nanaian — APPROVED** [review]
LGTM!

File: src/battle/area_flo2/5CE690.c [hunk 4]
```diff
@@ -18,9 +18,19 @@ INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_80218620_5CECB0);
 
 INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_80218774_5CEE04);
 
-INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_802188A8_5CEF38);
-
-INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_802188EC_5CEF7C);
+ApiStatus func_802188A8_5CEF38(ScriptInstance* script, s32 isInitialCall) {
+    func_8011D82C(1);
+    *D_801512F0 = 1;
```
Comment (ethteck) [inline src/battle/area_flo2/5CE690.c:23]: I forgot what D_801512F0 is defined as, but if it's an array, could you do D_801512F0[0] = 1 instead?

Comment (tashi-lp) [reply src/battle/area_flo2/5CE690.c:23]: It appears to be an `s8*`, and other functions define it directly

Comment (ethteck) [reply src/battle/area_flo2/5CE690.c:23]: Gotcha. Okay, nm

**Pull request merged**